OracleFMWInfrastructure/dockerfiles/12.2.1.3/run_admin_server.sh

```diff
@@ -19,19 +18,21 @@ set_context
 . ${scriptDir}/container-scripts/setEnv.sh ${scriptDir}/properties/domain.properties
 
 # HOST volume where the domain home will be persisted
-hostvolume=/home/username/domain_home
-echo "Host volume to write the domain is: $hostvolume"
+domain_host_volume() {
+
+   domainhostvol=${DOMAIN_HOST_VOLUME:-"/home/username/domain_home"}
```

Comment:
There doesn't appear to be a test to see if  ${DOMAIN_HOST_VOLUME} exists and the default of `/home/username/domain_home` is almost certainly _not_ going to exist. This seems (to me) to be the kind of option that shouldn't have a default because we have no idea where/how the user is going to run the script and should rather test for the existence of ${DOMAIN_HOST_VOLUME} and error out if it doesn't exist instead of trying to create a Docker container with a bind mount to something broken.

Reply:
The DOMAIN_HOME_VOLUME gets created when the container gets run if it does not already exist.  I want to reuse it if it was created in a previous run, that is if the admin server container comes down and I re-start I do not want the DOMAIN_HOME_VOLUME created again.  If DOMAIN_HOME_VOLUME is an invalid directory the container will fail to run, WebLogic will not configure the domain and it will throw an error. 
I will make changes to not take a default and make sure that the DOMAIN_HOME_VOLUME is set in the domain.properties file.

Reply:
Sounds good to me. :)